install

```diff
@@ -199,6 +192,17 @@ not intentional, please restore the default permissions and try running the
 installer again:
     sudo chmod 775 #{HOMEBREW_PREFIX}
 EOABORT
+# Clone using git:// if TLS 1.2 not built in.
```

Comment:
Pendatry: empty line above this one?

Reply:
Also: is there code missing heee? It doesn’t seem to match what the code immediately below does.

Reply:
I've updated the comment.